front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -79,38 +82,58 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
       id="project-ideas"
       className={`e2e-timeline-project-idea-cards ${className || ''}`}
     >
-      {!isVotingContext && (
-        <Title variant="h2" mt="0px" mb="20px" color="tenantText">
-          <FormattedMessage
-            {...getInputTermMessage(inputTerm, {
-              idea: messages.ideas,
-              option: messages.options,
-              project: messages.projects,
-              question: messages.questions,
-              issue: messages.issues,
-              contribution: messages.contributions,
-              proposal: messages.proposals,
-              initiative: messages.initiatives,
-              petition: messages.petitions,
-            })}
+      {isVotingContext ? (
+        <IdeaCardsWithoutFiltersSidebar
+          ideaQueryParameters={ideaQueryParameters}
+          onUpdateQuery={updateSearchParams}
+          className={participationMethod}
+          projectId={projectId}
+          showViewToggle={true}
+          defaultSortingMethod={ideaQueryParameters.sort}
+          defaultView={phase.attributes.presentation_mode}
+          invisibleTitleMessage={messages.a11y_titleInputsPhase}
+          phaseId={phase.id}
+          showDropdownFilters={false}
+          showSearchbar={false}
+        />
+      ) : (
+        <>
+          <Box
+            position="absolute"
+            mt="-100px"
+            id="ideas-list-scroll-anchor"
+            aria-hidden={true}
           />
```

Comment:
Making a component of this can also be a separate ticket/PR.

Reply:
https://www.notion.so/govocal/Made-ideas-list-scroll-anchor-its-own-component-1499663b7b26807ca1c9d8ab12b5d978?pvs=4